inc/order_item.class.php

```diff
@@ -275,28 +287,29 @@ public function checkIFReferenceExistsInOrder($orders_id, $ref_id) {
    }
 
 
-   public function addDetails($ref_id, $itemtype, $orders_id, $quantity, $price, $discounted_price, $taxes_id) {
+   public function addDetails($ref_id, $itemtype, $orders_id, $quantity, $price, $discounted_price, $taxes_id, $analytic_nature_id) {
```

Comment:
I guess calls on https://github.com/pluginsGLPI/order/blob/develop/front/order.form.php#L249 and https://github.com/pluginsGLPI/order/blob/develop/front/order.form.php#L268 have to be updated.